src/Tizen.NUI/src/internal/KeyboardRepeatSettingsChangedSignal.cs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
```

Comment:
please add some sample code or usage example from next time. ^^; 
need to know how this can be used even though it is hidden API...

Reply:
Okay, if I add a new API to NUI next time, I'll make a simple example.